cli/args/flags.rs

```diff
@@ -718,12 +693,25 @@ fn handle_repl_flags(flags: &mut Flags, repl_flags: ReplFlags) {
 }
 
 fn clap_root() -> Command {
+  let long_version = format!(
+    "{} ({}, {})\nv8 {}\ntypescript {}",
+    crate::version::deno(),
+    if crate::version::is_canary() {
+      "canary"
+    } else {
+      env!("PROFILE")
+    },
+    env!("TARGET"),
+    deno_core::v8_version(),
+    crate::version::TYPESCRIPT
+  );
```

Comment:
Ideally this would be a static string too, but that would require to change `deno_core::v8_version()` to acquire V8 version information during build time. Doable, but I don't have time for this now.